<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupGeneral">
     <property name="styleSheet">
      <string notr="true">QGroupBox {
	border: 1px solid #7F7F7F;
	border-radius: 3px;
	margin-top: 1ex;
 }

QGroupBox::title {
	subcontrol-origin: margin;
	subcontrol-position: top left; 
	padding: 0 2px;
 }</string>
     </property>
     <property name="title">
      <string>General</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="labelLimit">
        <property name="text">
         <string>Movies limit</string>
        </property>
        <property name="buddy">
         <cstring>comboLimit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboLimit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>30</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>50</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>70</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>100</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelCacheDays">
        <property name="text">
         <string>Days to keep cache</string>
        </property>
        <property name="buddy">
         <cstring>comboDays</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboDays">
        <property name="currentIndex">
         <number>3</number>
        </property>
        <item>
         <property name="text">
          <string>3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>30</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>90</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>180</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupSubtitles">
     <property name="styleSheet">
      <string notr="true">QGroupBox {
	border: 1px solid #7F7F7F;
	border-radius: 3px;
	margin-top: 1ex;
 }

QGroupBox::title {
	subcontrol-origin: margin;
	subcontrol-position: top left; 
	padding: 0 2px;
 }</string>
     </property>
     <property name="title">
      <string>Subtitles</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="labelLanguage">
        <property name="text">
         <string>Language</string>
        </property>
        <property name="buddy">
         <cstring>comboLanguage</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboLanguage">
        <property name="currentIndex">
         <number>12</number>
        </property>
        <item>
         <property name="text">
          <string>Albanian</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Arabic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Belarus</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bengali</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bosnian</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bulgarian</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Catalan</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Chinese</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Croation</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Czech</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Danish</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dutch</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>English</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Estonian</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Finnish</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>French</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>German</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Greek</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hebrew</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hindi</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hungarian</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Icelandic</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Indonesian</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Irish</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Italian</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Japanese</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Kazakh</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Korean</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Latvian</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Lithuanian</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Macedonian</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Malay</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Norwegian</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Polish</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Portuguese</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Romanian</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Russian</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Serbian</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Sinhala</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Slovak</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Slovenian</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Spanish</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Swedish</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Thai</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Turkish</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Ukranian</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Vietnamese</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelEncoding">
        <property name="text">
         <string>Codepage</string>
        </property>
        <property name="buddy">
         <cstring>comboCodepage</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboCodepage">
        <item>
         <property name="text">
          <string>Auto</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BIG-5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ISO_8859-1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ISO_8859-13</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ISO_8859-14</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ISO_8859-15</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ISO_8859-2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ISO_8859-3</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ISO_8859-4</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ISO_8859-5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ISO_8859-6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ISO_8859-7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ISO_8859-8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ISO_8859-9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>KOI8-R</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>KOI8-U</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>SHIFT_JIS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>UTF-16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>UTF-8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CP1250</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CP1251</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CP1253</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CP1256</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupTorrents">
     <property name="styleSheet">
      <string notr="true">QGroupBox {
	border: 1px solid #7F7F7F;
	border-radius: 3px;
	margin-top: 1ex;
 }

QGroupBox::title {
	subcontrol-origin: margin;
	subcontrol-position: top left; 
	padding: 0 2px;
 }</string>
     </property>
     <property name="title">
      <string>Torrents</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QLabel" name="labelDlRate">
        <property name="text">
         <string>Maximum download rate (KB/s)</string>
        </property>
        <property name="buddy">
         <cstring>comboDlRate</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="4">
       <widget class="QComboBox" name="comboDlRate">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>-1 = Unlimited</string>
        </property>
        <item>
         <property name="text">
          <string>-1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>50</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>100</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>500</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10000</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="9" column="4">
       <widget class="QPushButton" name="pushDlPath">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="4">
       <widget class="QLabel" name="labelPort">
        <property name="text">
         <string>Port for incoming connections</string>
        </property>
        <property name="buddy">
         <cstring>spinPort</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="checkKeepFiles">
        <property name="toolTip">
         <string>Keep completed files after exiting</string>
        </property>
        <property name="text">
         <string>Keep completed files </string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="checkEncryption">
        <property name="toolTip">
         <string>Enable protocol encryption (avoid ISP block)</string>
        </property>
        <property name="text">
         <string>Encryption</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLineEdit" name="lineDlPath">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QComboBox" name="comboUlRate">
        <property name="toolTip">
         <string>-1 = Unlimited</string>
        </property>
        <item>
         <property name="text">
          <string>-1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>50</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>100</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>500</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10000</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="4" column="4">
       <widget class="QSpinBox" name="spinPort">
        <property name="minimum">
         <number>6800</number>
        </property>
        <property name="maximum">
         <number>6999</number>
        </property>
        <property name="value">
         <number>6881</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelUlRate">
        <property name="text">
         <string>Maximum upload rate (KB/s)</string>
        </property>
        <property name="buddy">
         <cstring>comboUlRate</cstring>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="labelDlPath">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Download path:</string>
        </property>
        <property name="buddy">
         <cstring>lineDlPath</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>&amp;Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>Settings</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>383</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
